src/radio/sx126x/radio.c

```diff
@@ -1288,6 +1288,9 @@ void RadioIrqProcess( void )
             TimerStop( &TxTimeoutTimer );
             
             // This is the Tx Transmit Time
+            // The calls to the ODR module below dosen't follow the convention used elsewhere in this file

```

Comment:
Still not completely right:
```suggestion
            // The calls to the ODR module below doesn't follow the convention used elsewhere in this file

```